src/main/java/uk/gov/hmcts/reform/cpo/auditlog/aop/AuditContextHolder.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.cpo.auditlog.aop;
+
+public class AuditContextHolder {
+
+    private AuditContextHolder() {
```

Comment:
Could use Lombok `@UtilityClass` here to

- automatically generate a private constructor that throws an exception
- 
- flags as error any explicit constructors you add
- 
- marks the class final

Reply:
Not actioned as `UtilityClass` is still marked as experminetal: i.e. `lombok.experimental.UtilityClass`

Reply:
Good spot, I didn't see that.